pkg/deployments/tasks/20230427-gauge-working-balance-helper/readme.md

```diff
@@ -0,0 +1,9 @@
+# 2023-04-27 - Gauge Working Balance Helper
+
+Deployment of the `GaugeWorkingBalanceHelper`, which reads the current and projected `working_balance` (effective veBAL boosted balance) for a user on a gauge, as well as the ratio of the working_balance to the supply. This can be used (e.g., by front-end processes) to determine whether checkpointing a gauge would be advantageous.
+
+## Useful Files
+
+- [`GaugeWorkingBalanceHelper` artifact](./artifact/GaugeWorkingBalanceHelper.json)
+- [`LiquidityGaugeV5` artifact](./artifact/LiquidityGaugeV5.json)
+- [`ChildChainGauge` artifact](./artifact/ChildChainGauge.json)
```

Comment:
Why are these included? I'd remove them entirely

Reply:
The fork tests don't work without them